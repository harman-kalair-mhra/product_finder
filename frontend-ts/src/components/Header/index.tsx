import React from "react";
import styled from "styled-components";

const StyledH1 = styled.h1`
  color: blue;
  font-size: 30px;
  background-color: lightblue;
  border: 2px solid lightblue;
  border-radius: 30px;
`;
interface HeaderProps {
  agency: string;
}

export default function Header() {
  return <StyledH1>ORDER YOUR DRUGS HERE!</StyledH1>;
}
