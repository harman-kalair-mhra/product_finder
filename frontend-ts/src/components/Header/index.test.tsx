import React from "react";
import { shallow, mount } from "enzyme";

import Header from "./index";

describe("Header", () => {
  it("renders shallow", () => {
    const component = shallow(<Header agency="MHRA"></Header>);
    expect(component).toMatchSnapshot();
  }),
    it("renders mount", () => {
      const component = mount(<Header agency="MHRA"></Header>);
      expect(component).toMatchSnapshot();
    });
});
