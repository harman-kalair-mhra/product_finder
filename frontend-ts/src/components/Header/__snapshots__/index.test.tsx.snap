// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders mount 1`] = `
<Header
  agency="MHRA"
>
  <styled.h1>
    <h1
      className="sc-AxjAm bwvbkn"
    >
      Product Finder 
      MHRA
    </h1>
  </styled.h1>
</Header>
`;

exports[`Header renders shallow 1`] = `
<styled.h1>
  Product Finder 
  MHRA
</styled.h1>
`;
